README.md

```diff
@@ -1,14 +1,18 @@
-# Safe{Core} Protocol Specs
+# Safe{Core} Protocol Specification
+
+See also:
+- [Protocol Implementation](https://github.com/5afe/safe-core-protocol)
+- [Protocol Demo Application](https://github.com/5afe/safe-core-protocol-demo/)
 
 ## Motivation
 
 Safe{Core} Protocol is an open, modular framework to make smart accounts secure, portable, and composable.
 
-## Objective
+## Objectives
 
-The goal is to create a comprehensive system, designed to enhance the safety and flexibility of user interactions within the smart account ecosystem. The protocol provides a common framework for building additional tools, encourages the composition of various integrations, and supports all types of smart accounts. The protocol **enforces the correct conduct and procedures** of a system of rules of the **smart account ecosystem**.
+The goal is to create a comprehensive system, designed to enhance the safety and flexibility of user interactions within the smart account ecosystem. The protocol provides a common framework for building additional tools, encourages the composition of various integrations, and supports different types of smart accounts. In order to achieve interoperability and security, the protocol enforces a set of rules within the smart account ecosystem.
```

Comment:
should we bold `interoperability` and `security`.

You also mentioned `discoverability` before ... should we include this?

Reply:
Would not include this. At least not in connection with this point (enforcement of rules).